framework/permissions.py

```diff
@@ -61,28 +61,43 @@ def can_create_feature(user):
 
 
 def can_edit_any_feature(user):
-  """Return True if the user is allowed to edit features."""
-  return can_create_feature(user)
+  """Return True if the user is allowed to edit all features."""
+  if not user:
+    return False
+  app_user = user_models.AppUser.get_app_user(user.email())  
+  if not app_user:
+    return False
 
+  # Site editors or admins should be able to edit any feature.
+  return app_user.is_admin or app_user.is_site_editor
 
-def can_edit_feature(user, feature):
-  """Return True if the user is allowed to edit the given feature."""
-  # TODO(jrobbins): make this per-feature
-  if not can_view_feature(user, feature):
+
+def feature_edit_list(user):
+  """Return a list of features the current user can edit"""
+  if not user:
     return False
```

Comment:
False is not a list.

Reply:
Thanks @jrobbins. I was having trouble rebasing the changes and dealing with conflicts after all the recent changes, and I might have some issues here. I'm trying to find them all, but perhaps more will be found in manual testing.